src/test/java/edu/ucsb/cs156/organic/Interceptors/RoleUserInterceptorTests.java

```diff
@@ -1,5 +1,5 @@
 
-package edu.ucsb.cs156.organic.interceptors;
+package edu.ucsb.cs156.organic.Interceptors;
```

Comment:
NOOOOO!!! 

We already fixed this!
Don't bring back this bad code that is unrelated to your changes!